agent/ansible/pbench/agent/roles/pbench_agent_files_install/tasks/main.yml

```diff
@@ -2,37 +2,37 @@
 # Install file(s) for pbench agent.
 - name: create temp dir to store the file(s) locally
   delegate_to: localhost
-  become: no
```

Comment:
Why is `false` better than `no`?  Given that the positive value appears to be `yes`, I would think that we want to stay with `no` for the negative value.  Alternatively, perhaps we should just omit the field?  (The [docs](https://docs.ansible.com/ansible/latest/user_guide/become.html#become-directives) show no examples using anything other than `yes`.)

Reply:
Is this the result of the linting?

Reply:
`become: false` means no sudo privilege required
`False/No` or `True/Yes` could be interchangeably used [link](https://docs.ansible.com/ansible/2.4/become.html#directives).
Use of `True/False` is a best practice in YAML values, it is encouraged to use `True/False` instead of `Yes/No`.

Reply:
`ansible-lint` insists on `true/false` (and IIRC it does not like `True/False` either, so it seems to follow YAML 1.2)), but "truthy" values are rampant both in the docs and in the wild.